src/de/tests.rs

```diff
@@ -159,3 +159,16 @@ fn untagged() {
 
     assert_eq!(from_str::<Untagged>("true").unwrap(), Untagged::Bool(true));
 }
+
+#[test]
+fn forgot_apostrophes() {
+    let de: Result<(i32, String)> = from_str("(4, \"Hello)");
+
+    assert!({
+        if let Err(Error::Parser(ParseError::ExpectedStringEnd, _)) = de {
```

Comment:
match ftw!

Reply:
I wondered why this isn't easier, but how does match help here?

Reply:
(He might be referencing the concept of pattern matching?)

Reply:
I just don't see the point in `if let` sugar over `match` if you take both arms and occupy more lines in the end